pkg/apis/api.acorn.io/v1/types.go

```diff
@@ -96,11 +96,11 @@ type Image struct {
 	metav1.TypeMeta   `json:",inline"`
 	metav1.ObjectMeta `json:"metadata,omitempty" protobuf:"bytes,1,opt,name=metadata"`
 
-	Digest      string `json:"digest,omitempty"`
-	Repository  string `json:"repository,omitempty"`
-	Tag         string `json:"tag,omitempty"`
-	Reference   string `json:"reference,omitempty"`
-	ClusterName string `json:"clusterName,omitempty"`
+	Digest      string   `json:"digest,omitempty"`
+	Repository  string   `json:"repository,omitempty"`
```

Comment:
Based on our previous convo, this field and the Reference field need completely dropped from this struct.